content/docs/user-guide/data-management/storage-locations.md

```diff
@@ -0,0 +1,38 @@
+# Storage locations
```

Comment:
Not sure this needs its own page, but just noticed it is not included in the sidebar.